### PR TITLE
Ok fuck it, time to nerf mining

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

TLDR makes the 3 prominent megafauna suits actually suitable for mining and basically nothing else.
Ditches the spellblade because it is a joke, fully replacing it with the crucible.

**Praetor:**
armor = list("melee" = 80, "bullet" = 20, "laser" = 20, "energy" = 30, "bomb" = 70, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)

**HECK:**
armor = list("melee" = 70, "bullet" = 20, "laser" = 10, "energy" = 25, "bomb" = 60, "bio" = 60, "rad" = 50, "fire" = 100, "acid" = 100)

**Dragonbone:**
armor = list("melee" = 70, "bullet" = 10, "laser" = 20, "energy" = 25, "bomb" = 60, "bio" = 60, "rad" = 50, "fire" = 100, "acid" = 100)

## Why these stats make sense

Miners basically face three threats: Explosions (from PKA blasts and gibtonite), fire (from ash storms and imps) and of course melee. This basically makes them god tier at these things at the expense of everything else, thus giving powerminers a clear way to be beaten even if they have the greatest loot.

**Counter-arguments:**
Colossus death bolts: Not affected by armor. They always 100% penetrate.
Watcher bolts: Changes internal temperature of the mob. Armor does not matter.
Hierophant: Completely ignores armor.
Elites: None of them have laser or bullet attacks (that account armor).

## Why It's Good For The Game

i hate mining 😠 

## Changelog
:cl:
balance: Spellblade has been fully replaced with the crucible sword.
balance: Nerfed the praetor suit.
balance: Nerfed the HECK suit.
balance: Nerfed the ash drake cloak.
/:cl:

### CODE REVIEW

**Comment by Jamini** [comment]
Slight typo in the values in the description vs the PR

HECK and Drake have 60 Bomb resist in this PR
Praetor has 70 Bomb resist in this PR

Considering that neither security nor most antagonists will normally have weapons with bomb damage types, and the only antagonists that regularly explode (Kudzu and Blob types) are expected to have miners go at them anyway. This shouldn't be an issue. (Minus the edge case of a miner antag using a KA. But who really cares about miner on miner violence anyway?)

**Comment by BobJoga** [comment]
> Slight typo in the values in the description vs the PR
> 
> HECK and Drake have 60 Bomb resist in this PR
> Praetor has 70 Bomb resist in this PR
> 
> Considering that neither security nor most antagonists will normally have weapons with bomb damage types, and the only antagonists that regularly explode (Kudzu and Blob types) are expected to have miners go at them anyway this shouldn't be an issue. (Minus the edge case of a miner antag using a KA. But who really cares about miner on miner violence anyway?)

Oh woops. Fixing those.

File: modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -351,7 +351,7 @@
 /obj/item/clothing/suit/space/hardsuit/deathsquad/praetor
 	name = "Praetor Suit"
 	desc = "And those that tasted the bite of his sword named him... The Doom Slayer."
-	armor = list("melee" = 75, "bullet" = 55, "laser" = 55, "energy" = 45, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
+	armor = list("melee" = 80, "bullet" = 20, "laser" = 20, "energy" = 30, "bomb" = 70, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
```
Comment (ForrestWick) [inline modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm:354]: Bullet, laser, and energy shouldn't be the same. If you make one high, you have to make the other low. This is basic balance. if it's the same thing, with melee being absolutely insane, then the ways to counter it are then nullified.

Comment (BobJoga) [reply modular_skyrat/code/modules/mining/lavaland/necropolis_chests.dm:354]: 20 is low. These are not good stats and bullets and laser can still easily counter the suit. Same for energy. Stop trying to move the goalpost.

**Comment by calibraptor** [comment]
In support of this. Mining has long since experienced powercreep on /tg/. It's time we start to roll it back to reasonable levels.
